Source/PLATEAUEditor/Private/ExtentEditor/PLATEAUFeatureInfoDisplay.cpp

```diff
@@ -212,6 +212,13 @@ FString FPLATEAUFeatureInfoDisplay::GetIconFileName(const PredefinedCityModelPac
     }
 }
 
+/**
+ * @brief 範囲選択画面に表示する順番でアイコンファイル名リストを取得
+ */
+TArray<FString> FPLATEAUFeatureInfoDisplay::GetIconFileNameList() {
+    return TArray<FString> { "building.png", "plants.png", "props.png", "traffic.png", "underground.png", "bridge.png", "terrain.png", "other.png" };
```

Comment:
細かく指定してなかったので恐縮ですが、重要度順にしたいので以下の並びでお願いします！
また、ファイル名複数個所で同じ文字列使用しているので抽象化お願いします。
```suggestion
    return TArray<FString> { "building.png", "traffic.png", "props.png", "bridge.png", "plants.png", "underground.png", "terrain.png", "other.png" };
```

Reply:
解決確認しました